easybuild/easyconfigs/b/bokeh/bokeh-3.2.0-foss-2022b.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'PythonBundle'
+
+name = 'bokeh'
+version = '3.2.0'
+
+homepage = 'https://github.com/bokeh/bokeh'
+description = "Statistical and novel interactive HTML plots for Python"
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+builddependencies = [
+]
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('PyYAML', '6.0'),
+    ('Pillow', '9.4.0'),
+    ('SciPy-bundle', '2023.02'),
+    ('meson-python', '0.11.0'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('tornado', '6.3.2', {
+        'checksums': ['4b927c4f19b71e627b13f3db2324e4ae660527143f9e1f2e2fb404f3a187e2ba'],
+    }),
+    ('contourpy', '1.0.7', {
+        'checksums': ['d8165a088d31798b59e91117d1f5fc3df8168d8b48c4acc10fc0df0d0bdbcc5e'],
+    }),
+    ('xyzservices', '2023.7.0', {
+        'checksums': ['0ec928742227d6f5d4367ea7b457fcfed943429f4de2949b5b02a82cdf5569d6'],
+    }),
+    ('setuptools', '67.8.0', {
+        'checksums': ['62642358adc77ffa87233bc4d2354c4b2682d214048f500964dbe760ccedf102'],
+    }),
+    (name, version, {
+        'preinstallopts': """sed -i 's/setup(/setup(version="%(version)s",/g' setup.py && """,
```

Comment:
This looks to be new and clearly fixes something in the sources. Can you include a comment what was wrong here that needed to be fixed?